### PR TITLE
BLorient587

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient587.xml [hunk 1]
```diff
@@ -38,57 +38,68 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                            
                         </handDesc>   
+                        
                         <additions>
-                            
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#67r"/>
+                                    <desc>A Patriarch John is mentioned in a prayer, maybe
+                                        to be identified with Pope <persName ref="PRS5713JohnXVI"/> (r. 1769-1796).</desc>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient587.xml:85]: "r." is good for kings etc., for a patriarch I would suggest "in tenure".

File: LondonBritishLibrary/orient/BLorient587.xml [hunk 2]
```diff
@@ -38,57 +38,68 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                             </handNote>
-                            
                         </handDesc>   
+                        
                         <additions>
-                            
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#67r"/>
+                                    <desc>A Patriarch John is mentioned in a prayer, maybe
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient587.xml:84]: "A Patriarch John" looks a little strage (I understand what is meant, though), sure that this constellation is possible?

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient587.xml (hunk 2 above)
Comment (eu-genia) [reply LondonBritishLibrary/orient/BLorient587.xml:84]: If you mean English, the English is correct. means "A certain ...."

**Review by eu-genia — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient587.xml (hunk 2 above)
Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient587.xml:84]: "Patriarch John" is from Wright, the hypothesis from me, I will be happy for a better solution!

Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/orient/BLorient587.xml:84]: I would propose a small change: "A patriarch John".

**Pull request merged**